.env

```diff
@@ -3,3 +3,6 @@ CELERY_BROKER_URL=redis://redis
 CELERY_RESULT_BACKEND=redis://redis
 API_URL=http://api/api
 API_BASE_URL=http://api
+
```

Comment:
I would move this to another env file `.env-legends`to be included in the `docker-compose.yml` file